packages/cli/package.json

```diff
@@ -7,15 +7,16 @@
   "license": "MIT",
   "scripts": {
     "copy-files": "./scripts/copy-files.sh",
-    "compile:ts": "rm -rf lib && tsc",
+    "compile:ts": "tsc",
     "compile:contracts": "rm -rf build/contracts && ../../bootstrap/node_modules/.bin/oz compile --solc-version 0.5.14 --evm-version constantinople",
     "prepare": "npm run compile:contracts && npm run compile:ts && npm run copy-files && chmod 755 ./lib/bin/oz-cli.js",
     "prepare-docs": "TS_NODE_PROJECT='tsconfig.docs.json' ts-node scripts/prepare-docs.js",
-    "test": "TS_NODE_PROJECT='tsconfig.test.json' mocha --require ts-node/register --recursive test --extension js,ts",
+    "test": "tsc -p tsconfig.test.json && mocha --recursive test-build/test",
```

Comment:
If this would be important for your workflow, sure. However, I'm slightly reluctant to add so many scripts, such that it would be hard to know which one to use. I think in the scenario you're imagining one could simply use `yarn test`, which has a compilation overhead of about 4 seconds.